2025/selection-process.md

```diff
@@ -0,0 +1,81 @@
+
+In this stage, the focus would be on organizations picking areas where there is internal alignment on the specific feature or area being important enough to work on from a user/web developer perspective. Each organization will have different rubrics for prioritization. However, where applicable, organizations can choose to have any data used for prioritization be shared publicly.
+
+3. Each organization will select proposals for the prioritization discussion by putting a High/Low priority signal against each proposal where they have internal alignment to do the work. A low priority signal would indicate a willingness to support the proposal, if other organizations also deem it to be important. Organizations can also express any objections at this point.
+4. Any proposal that has at least 2 organizations expressing high priority and no objections, would be moved forward to the next phase of consensus building. Any organization that posts an objection would also outline the reasoning for it, to the Interop team. The ranking of proposals would be done asynchronously, ahead of the Interop meetings.
+
+
+### Phase 3: Prioritization (Nov 1st to Dec 5th)
+
+Interop meeting dates: Nov 7th, Nov 14th, Nov 21st, Dec 5th
+
+[Visibility] All decisions made in this phase will be confidential. 
+
+In this stage, each organization will further prioritize proposals for a specific year (Interop 2025). This is more involved and a complex process where each organization weighs engineering priorities and available resources.
+
+5. From the list of proposals that get through to Phase 3, each organization will have an opportunity to further prioritize internally and build consensus within the Interop Team based on available signals.
+6. Organizations can also object to any proposal at this point.
+7. Any proposal that has support from 2 organizations and does not have an objection, will be considered included in Interop 2025. At the end of this phase, the expectation is that the Interop team will have a first draft of proposals that are included in Interop 2025.
+
+
+### Phase 4: Carryovers and Grouping (Dec 6th to Dec 19th)
+
+Interop meeting dates: Dec 12th and Dec 19th
+
+[Visibility] All decisions and results from this phase will be confidential.
+
+In this stage, the Interop team will evaluate progress made in Interop 2024 focus areas and create a first draft of areas that will be carried over to Interop 2025. In addition, the proposals from phase 3 will be weighed against any carryover focus areas from Interop 2024. The outcome from this phase is a second draft of prioritized proposals for Interop 2025 (this will include new proposals and carryover focus areas) with a high-level grouping. This stage will include the steps below:
+
+8. Interop 2024 carryover evaluation - this will be done live in the Interop team meeting.
```

Comment:
This should be clearer about how carryover evaluation works. If there is no consensus on whether or not to carryover a given proposal, is it default-carryover or default-discard?